packages/boxel/addon/components/boxel/drop-target/index.gts

```diff
@@ -0,0 +1,33 @@
+import type { TemplateOnlyComponent } from '@ember/component/template-only';
+import { type EmptyObject } from '@ember/component/helper';
+import cn from '@cardstack/boxel/helpers/cn';
+import { concat } from '@ember/helper';
+import { svgJar } from '@cardstack/boxel/utils/svg-jar';
+
+import '@cardstack/boxel/styles/global.css';
+import './index.css';
+
+export interface Signature {
+  Element: HTMLDivElement;
+  Args: {
+    state?: string;
+  };
+  Blocks: EmptyObject;
+}
+
+const DropTarget: TemplateOnlyComponent<Signature> = <template>
```

Comment:
I was wondering if there's some sort of pattern for closing and opening template tags.

for example,  add `<template>`  the way it is right after the `=` sign, or add it after skipping a line.

Reply:
This format is new enough that there are not community standards yet AFAIK. I have been doing <template> after the equal sign for template only components and after a blank line for class components.